libs/estdlib/src/gen_tcp.erl

```diff
@@ -59,6 +59,8 @@
 -type connect_option() :: option().
 -type packet() :: string() | binary().
 
+-export_type([reason/0, option/0, listen_option/0, connect_option/0, packet/0]).
```

Comment:
I'm trying to understand why these types need to be exported. Are they used as types in other modules? I don't see that they are being used by ssl.erl.

Reply:
they might be used from other applications around. That's how they are defined in otp.